packages/dds/cell/src/interfaces.ts

```diff
@@ -42,3 +42,13 @@ export interface ISharedCell<T = any> extends ISharedObject<ISharedCellEvents<T>
      */
     delete(): void;
 }
+
+export interface ICellValue {
```

Comment:
I don't think these interfaces should be exported since they are internal implementation details to cell. Do you just need them for tests? I think you can get around that.